### PR TITLE
Added ServerFlex to Java Version page

### PR DESCRIPTION
**Pull request by alandoherty**
Added ServerFlex to Java 16 version page as we still have some confused customers trying to update manually and this is the first result on Google. Let me know if I need to make any changes.

### CODE REVIEW

File: source/java-update/index.rst [hunk 1]
```diff
@@ -138,6 +138,15 @@ PebbleHost's knowledgebase has a great article `Does PebbleHost support Java 16?
 on their website to show how to change the Java version, along with
 incompatibilities with certain versions.
 
+ServerFlex
+~~~~~~~~~~
+
+ServerFlex defaults to Java 8, but will automatically select Java 16 where appropriate. You can configure the
+java version manually at the bottom of the server settings page, you can also find more information `on the support article.
+<https://serverflex.io/help/article/change-java-version-on-minecraft-server>`_
```
Comment (sulu5890) [inline source/java-update/index.rst:146]:
```suggestion
ServerFlex defaults to Java 8, but will automatically select Java 16
where appropriate. To manually configure the java version, navigate
to the settings page, and select ``Java Version``. More information can be found
`in ServerFlex's support article. <https://serverflex.io/help/article/change-java-version-on-minecraft-server>`_
```

**Review by sulu5890 — CHANGES_REQUESTED** [review]
Thank you for adding this, just a small change to better fit the style of this page.

File: source/java-update/index.rst [hunk 2]
```diff
@@ -138,6 +138,15 @@ PebbleHost's knowledgebase has a great article `Does PebbleHost support Java 16?
...
+
+.. image:: java-update-assets/serverflex-java-version.png
```
Comment (Proximyst) [inline source/java-update/index.rst:148]: All images have to be prefixed a new line with `|br|`:
```suggestion
.. image:: java-update-assets/serverflex-java-version.png

|br|
```

**Review by Proximyst — CHANGES_REQUESTED** [review]

**Comment by alandoherty** [comment]
Applied changes, looks cleaner now thanks!

**Review by Oharass — APPROVED** [review]

**Review by Proximyst — APPROVED** [review]

**Pull request merged**